.github/workflows/backport.yml

```diff
@@ -16,8 +16,8 @@ permissions: {}
 
 jobs:
   main:
-    # skip it in all forks, only run in grafana/tempo.
-    if: github.repository == 'grafana/tempo'
+    # skip it in all forks and only run it for merged PRs
```

Comment:
fyi: if we only run it in merged PRs, it will not comment "PR needs to be merged for backport" on the PRs where we apply the label on an unmerged PR.

Reply:
reverted